.flake8

```
@@ -1,7 +1,10 @@
 [flake8]
 # Rule definitions: http://flake8.pycqa.org/en/latest/user/error-codes.html
 # W503: line break before binary operator
+# In most cases Black follows the rules below, but there are some exceptions
+# E203: whitespace before ':'
+# E501: line too long
 exclude = venv*,__pycache__,node_modules,cache
-ignore = W503
+ignore = W503, E203, E501
 max-complexity = 14
 max-line-length = 130
```

```suggestion
max-line-length = 88
```
To be consistent with `black`

---

I set the black line length to be 130 over here: https://github.com/cds-snc/notification-admin/blob/experiment/adding-black/pyproject.toml#L2. 

But I can change them both to be 88 if you prefer.

---

Sounds good, sorry I missed this, many files were changed 😅

---

Haha, yes this file was buried in the 182 other files that were changed.